src/main/java/com/google/api/codegen/transformer/php/PhpSurfaceNamer.java

```diff
@@ -124,6 +125,11 @@ public String getFullyQualifiedApiWrapperClassName(GapicInterfaceConfig interfac
   }
 
   @Override
+  public String getApiWrapperClassImplName(InterfaceConfig interfaceConfig) {
+    return publicClassName(Name.upperCamel(getInterfaceName(interfaceConfig), "Gapic"));
```

Comment:
I feel "Gapic" does not reflect what the class is, maybe something like "ClientImp" ?